airbyte-integrations/connectors/destination-jdbc/src/main/java/io/airbyte/integrations/destination/jdbc/copy/CopyDestination.java

```diff
@@ -92,4 +93,11 @@ public AirbyteConnectionStatus check(final JsonNode config) {
     }
   }
 
+  protected void performCreateInsertTestOnDestination(final String outputSchema,
```

Comment:
Since there was [discussion to separate this method temporarily](https://airbytehq-team.slack.com/archives/C03C4AVJWG4/p1667851893360659) so that you can narrow scope to only `destination-snowflake` for now, can you create a github issue and add it as a javadoc with a todo comment?

Reply:
This keeps the default state for all connectors except the Snowflake. In snowflake it's overridden to use a custom one. I will also mention this new ticket. Thanks